stdlib/2and3/time.pyi

```diff
@@ -1,29 +1,26 @@
-# Stubs for time
-# Ron Murawski <ron@horizonchess.com>
-
-# based on: http://docs.python.org/3.3/library/time.html#module-time
-# see: http://nullege.com/codes/search?cq=time
+"""Stub file for the 'time' module."""
+# See https://docs.python.org/3/library/time.html
 
 import sys
-from typing import Any, NamedTuple, Tuple, Union
-from types import SimpleNamespace
+from typing import Any, NamedTuple, Tuple, Union, Optional
+if sys.version_info >= (3, 3):
+    from types import SimpleNamespace
 
 TimeTuple = Tuple[int, int, int, int, int, int, int, int, int]
```

Comment:
This should be private.

Reply:
Agreed, the old Python 2 stub had it private, the Python 3 stub has it public. I wasn't sure whether there wasn't a reason for it to be public. But I can't really think of one.